.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,19 +1,15 @@
 ## Description
 
-Fixes (issue #)
```

Comment:
Personally, I forget to link issues if I don't see this.

Reply:
I actually add this to the commented block below.